library/src/main/java/com/squareup/timessquare/CalendarPickerView.java

```diff
@@ -387,6 +392,10 @@ public FluentInitializer displayOnly() {
     }
   }
 
+  public LinearLayoutManager getLinearLayoutManager() {
```

Comment:
this seems like a big implementation detail leak.  instead of exposing the layout manager, can you just expose a `findViewAtPosition(int)` method?